pkg/virt-operator/application.go

```diff
@@ -395,5 +395,13 @@ func (app *VirtOperatorApp) AddFlags() {
 }
 
 func (app *VirtOperatorApp) prepareCertManagers() {
-	app.operatorCertManager = bootstrap.NewFallbackCertificateManager(bootstrap.NewFileCertificateManager("/etc/virt-operator/certificates/tls.crt", "/etc/virt-operator/certificates/tls.key"))
+	app.operatorCertManager = bootstrap.NewFallbackCertificateManager(
+		[]certificate.Manager{
+			bootstrap.NewSecretCertificateManager(
+				components.VirtOperatorCertSecretName,
+				app.operatorNamespace,
+				app.informers.Secrets.GetStore(),
+			),
```

Comment:
not sure why this needs to be a list now

Reply:
A left over from another intial idea. Reverted.